src/scenes/CongratulationScene.ts

```diff
@@ -18,7 +18,7 @@ export default class CongratulationScene extends Scene {
             `You watched ${this.dataStore.score} seconds of TV!`,
             {align: 'center', fontSize: 42, fontFamily: 'Arial'}
         );
-        score.x = 768;
+        score.x = 750;
```

Comment:
Not really important, but these 768's were "half of 1536 width", not matching the height. These x values should be 812.